phoenix-core/src/test/java/org/apache/phoenix/schema/MetaDataClientTest.java

```diff
@@ -83,4 +85,13 @@ public void testHandleCreateTableMutationCodeWithNewCode() throws SQLException {
         }
     }
 
+    @Test
+    public void testMetaDataProtocolVersion() {
+        long serversTimstamp = System.currentTimeMillis();
+        String version = MetaDataProtocol.getVersion(serversTimstamp);
+        assertEquals("4.16.x", version);  // We have to update it after every release
+        assertEquals(String.format("%d.%d.%s", PHOENIX_MAJOR_VERSION, PHOENIX_MINOR_VERSION, "x"),
```

Comment:
I believe this assert might be better fit for all branches and the one above `assertEquals("4.16.x", version);` might not be needed. Thought?

Reply:
Yeah, that is right!